pyproject.toml

```diff
@@ -85,11 +72,18 @@ select = [
   "RUF", # Ruff-specific rules
   "I001", # isort
 ]
-fixable = ["I001"]
+fix = true
```

Comment:
I think this should not be here; instead, we should have `--fix` in `args` in `.pre-commit-config.yaml
```suggestion
```

Reply:
Ok, I will going to do that in a bit, thanks!

Reply:
it seems we don't need that for ruff format :)
so it would be totally fine to just remove that from pyproject.
thanks for caching that!